src/azmi-commandline/azmi-commandline.cs

```diff
@@ -201,7 +244,7 @@ static RootCommand ConfigureArguments()
             Operations operations = new Operations();
 
             // gettoken
-            getTokenCommand.Handler = CommandHandler.Create<string, string, bool, bool>((endpoint, identity, verbose, JWTFormat) =>
+            getTokenCommand.Handler = CommandHandler.Create<string, string, bool, bool>((endpoint, identity, JWTFormat, verbose) =>
```

Comment:
Is this change intended or just our test?

Reply:
Intended. Order of parameters should go from most important ones to least important in methods.